aptos-move/aptos-vm/src/transaction_metadata.rs

```diff
@@ -71,9 +71,9 @@ impl TransactionMetadata {
                 TransactionPayload::Script(s) => (s.code().len() as u64).into(),
                 _ => NumBytes::zero(),
             },
-            is_keyless: aptos_types::keyless::get_authenticators(txn)
-                .map(|res| !res.is_empty())
-                .unwrap_or(false),
+            num_keyless_authenticators: aptos_types::keyless::get_authenticators(txn)
+                .map(|res| res.len())
```

Comment:
Do we check somewhere number of authenticators fits into a u8? The cast can panic if not